ethereum/slogging.py

```diff
@@ -21,72 +21,19 @@
 eth.chain
 eth.chain.new_block
 """
+#!---------------
 
+FORMAT = "%(levelname)s:%(module)s %(asctime)-15s %(message)s"
 
 def hexprint(x):
     return repr(x)
 
-
-class KeyValueRenderer(structlog.processors.KeyValueRenderer):
-
-    """
-    Render `event_dict` as a list of ``Key=repr(Value)`` pairs.
-    Prefix with event
-    """
-
-    def __call__(self, _, __, event_dict):
-        msg = event_dict.pop('event', '')
-        kvs = ' '.join(k + '=' + hexprint(v) for k, v in self._ordered_items(event_dict))
-        return "%s\t%s" % (msg, kvs)
-
-
-class JSONRenderer(structlog.processors.JSONRenderer):
-
-    "JSON Render which prefixes namespace"
-
-    def __call__(self, logger, name, event_dict):
-        event_dict = dict(event_dict)
-        event_dict['event'] = logger.name + '.' + event_dict['event'].lower().replace(' ', '_')
-        return json.dumps(event_dict, cls=structlog.processors._JSONFallbackEncoder,
-                          **self._dumps_kw)
-
-
-######## TRACE ##########
-
-class BoundLoggerTrace(structlog.stdlib.BoundLogger):
-
-    "adds trace"
-
-    def trace(self, event=None, **kw):
-        """
-        Process event and call ``Logger.trace()`` with the result.
-        """
-        return self._proxy_to_logger('trace', event, **kw)
-
-    def is_active(self, level_name='trace'):
-        """
-        this is not faster, than logging.
-        can be used set flag in vm context
-        """
-        # any listeners?
-        if self._processors and isinstance(self._processors[0], LogListeners) \
-                and self._processors[0].listeners:
-            return True
-        # log level filter
-        return self._logger.isEnabledFor(structlog.stdlib._NAME_TO_LEVEL[level_name])
-
-
-structlog.stdlib.TRACE = TRACE = 5
-structlog.stdlib._NAME_TO_LEVEL['trace'] = TRACE
+TRACE = 5
 logging.addLevelName(TRACE, "TRACE")
```

Comment:
Please consolidate adding the `TRACE` level in one place. The rest of this seems to be in line 245+